Formula/musepack.rb

```diff
@@ -33,7 +33,11 @@ class Musepack < Formula
   def install
     system "cmake", ".", *std_cmake_args
     system "make", "install"
-    lib.install "libmpcdec/libmpcdec.dylib"
+    if OS.mac?
+      lib.install "libmpcdec/libmpcdec.dylib"
+    else
+      lib.install "libmpcdec/libmpcdec.so"
+    end
```

Comment:
```suggestion
    lib.install "libmpcdec/#{shared_library("libmpcdec")}"
```
Here's a more compact way of writing this.

It's not enough, unfortunately: https://github.com/Homebrew/homebrew-core/pull/92041